### DISCUSSION TITLE
OpenInv Integration with command blocks

### DISCUSSION THREAD

**Discussion by jedijacob2210** [Jikoo/OpenInv#185] [Q&A]
Is there a way to make OpenInv cooperate with command blocks? Eg. Search a player's inventory then execute a command?

**Comment by Jikoo** [comment]
Vanilla already has this capability.

If you're looking to chain the command block into another command block you can use [/clear](https://minecraft.fandom.com/wiki/Commands/clear) with an amount of 0, which outputs the amount of matching items as the success count rather than clearing anything.

If you're looking to conditionally execute a command directly, you can use [/execute if entity](https://minecraft.fandom.com/wiki/Commands/execute#ifunlessentity) conditionally [dependent on NBT](https://minecraft.fandom.com/wiki/NBT_format#Testing_NBT_tags), specifically the Inventory [tag](https://minecraft.fandom.com/wiki/Player.dat_format#NBT_structure). I found several promising results searching "minecraft command block check player inventory" if you're after a relatively preformed command. I'm not confident enough in my understanding of the execute command to write you a sample from scratch.
